packit_service/config.py

```diff
@@ -82,6 +83,9 @@ def __init__(self):
         # path to a file where OGR should store HTTP requests
         self.github_requests_log_path: str = ""
 
+        # for flask SERVER_NAME so we can create links to logs
+        self.server_name: str = ""
```

Comment:
Will you update [deployment/secrets/template/packit-service.yaml](https://github.com/packit-service/deployment/blob/master/secrets/template/packit-service.yaml)?

Reply:
+1 good point